Samples/dir.targets

```diff
@@ -5,10 +5,24 @@
   </PropertyGroup>
   
   <Import Project="..\dir.targets" />
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets"/>
 
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" Condition="'$(NetCoreBuild)' != 'true'"/>
-  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" Condition="'$(NetCoreBuild)' == 'true'"/>
+  <Import Project="$(NugetIntegrationDir)\Microsoft.NuGet.targets" Condition="'$(OsEnvironment)'=='Unix'" />
 
+  <Target Name="ResolveNuGetPackageAssets" Condition="exists('$(ProjectLockFile)')">
```

Comment:
Can you change this so it only overrides when building on Unix?  The task DLL in the NuGet.MsBuild.Integration package is test or delay signed, so I get a strong name validation error when I try to build using it on Windows.